### PR TITLE
Feat/rishabh winner history

### PR DESCRIPTION
**Pull request by rishxbh81**
## Description

winner history component is ready

## Related Issue(s)

If applicable, please link any issues that this PR addresses (e.g., `Closes #123`, `Fixes #456`).

- Issue(s) resolved: [link issues here]

## Type of Change

Please select the appropriate category for this PR:

- [ ] **Bug Fix** – Non-breaking changes that resolve issues
- [x] **New Feature** – Additions or enhancements to functionality
- [ ] **Refactor** – Code changes that improve readability or maintainability
- [ ] **Documentation** – Updates or additions to documentation
- [ ] **Tests** – Changes related to test coverage or testing strategy
- [ ] **Chore** – Other changes (e.g., maintenance, dependency updates)

## Checklist

Please make sure the following steps are completed before submitting your PR:

- [x] I have tested the code changes locally.
- [ ] I have update/added the documentation/`readme.draft.md` (if applicable).
- [ ] I have added/updated tests (if applicable).
- [x] My changes do not break existing functionality.
- [ ] I have rebased the branch with the latest version of the `development` branch.
- [x] The PR title is descriptive and follows the project's naming conventions.
- [x] The PR description is clear and includes all necessary information.

## Reviewer(s)

Please tag the appropriate reviewers for this PR:

- @Danishan2

### CODE REVIEW

File: src/components/GameComponents/PlaceStake/PlaceStake.jsx
Comment (Danishan2) [inline src/components/GameComponents/PlaceStake/PlaceStake.jsx:1]: This File has nothing

File: src/components/GameComponents/WinnerHistory/WinnerBox.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import style from "./style/WinnerHistory.module.css";
+
+const WinnerBox = ({ winner }) => {
```
Comment (Danishan2) [inline src/components/GameComponents/WinnerHistory/WinnerBox.jsx:3]: Refactored component to accept `winnerName`, `roundId`, and `onClick` as props for better reusability and dynamic rendering.

File: src/components/GameComponents/WinnerHistory/WinnerHistory.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import WinnerRectangle from "./WinnerRectangle";
+import style from "./style/WinnerHistory.module.css";
+
+const WinnerHistory = () => {
```
Comment (Danishan2) [inline src/components/GameComponents/WinnerHistory/WinnerHistory.jsx:5]: Refactored component to accept `GameId`, as props for better dynamic rendering.

**Review by Danishan2 — CHANGES_REQUESTED** [review]
Complete the suggested Changes

**Review by Danishan2 — APPROVED** [review]

**Review by Danishan2 — APPROVED** [review]

File: src/components/GameComponents/WinnerHistory/WinnerBox.jsx (hunk 1 above)
Comment (rishxbh81) [reply src/components/GameComponents/WinnerHistory/WinnerBox.jsx]: changes has been made

File: src/components/GameComponents/WinnerHistory/WinnerHistory.jsx (hunk 2 above)
Comment (rishxbh81) [reply src/components/GameComponents/WinnerHistory/WinnerHistory.jsx]: done

**Pull request merged**